api/filter/network/tcp_proxy.proto

```diff
@@ -1,6 +1,7 @@
 syntax = "proto3";
 
 package envoy.api.v2.filter.network;
+option go_package = "network";
```

Comment:
doesn't this option need to be added to all protos?

Reply:
One source is enough since we compile protos as a group. It's a side effect that `package == directory` in both golang and bazel. If we want fine-grained packages, we probably need fine-grained BUILD files.